lib/tasks/demo.rake

```diff
@@ -107,17 +167,22 @@ namespace :demo do
       end
     end
 
+    def reviewer
```

Comment:
It is created when we run seed rake mentioned in the demo rake but I think its safe to have `User.where(permission_id: 3).sample||FactoryGirl.create(:user, :reviewer)` rather than just `User.where(permission_id: 3).sample`. 
Thanks @steveyken for pointing this.